webtau-core/src/test/groovy/com/twosigma/webtau/expectation/code/ThrowExceptionMatcherGroovyTest.groovy

```diff
@@ -20,94 +20,95 @@ import org.junit.Rule
-                "\n" +
-                "expected exception message:   actual: error message <java.lang.String>\n" +
-                "                            expected: error message1 <java.lang.String>")
+        thrown.expectMessage('\nmismatches:\n' +
+                '\n' +
+                'expected exception message:   actual: "error message" <java.lang.String>\n' +
+                '                            expected: "error message1" <java.lang.String>')
 
         code {
-            throw new RuntimeException("error message")
-        } should throwException("error message1")
+            throw new RuntimeException('error message')
+        } should throwException('error message1')
     }
 
     @Test
     void "should validate exception message using regexp"() {
-        thrown.expectMessage("\nmismatches:\n" +
-                "\n" +
-                "expected exception message:    actual string: error message\n" +
-                "                            expected pattern: error \\d+")
+        thrown.expectMessage('\nmismatches:\n' +
+                '\n' +
+                'expected exception message:    actual string: error message\n' +
+                '                            expected pattern: error \\d+')
 
         code {
-            throw new RuntimeException("error message")
+            throw new RuntimeException('error message')
         } should throwException(~/error \d+/)
     }
 
     @Test
     void "should validate exception class"() {
-        thrown.expectMessage("\nmismatches:\n" +
-                "\n" +
-                "expected exception class:   actual: class java.lang.IllegalArgumentException <java.lang.Class>\n" +
-                "                          expected: class java.lang.UnsupportedOperationException <java.lang.Class>")
+        thrown.expectMessage('\nmismatches:\n' +
+                '\n' +
+                'expected exception class:   actual: class java.lang.IllegalArgumentException <java.lang.Class>\n' +
+                '                          expected: class java.lang.UnsupportedOperationException <java.lang.Class>')
 
         code {
-            throw new IllegalArgumentException("error message")
+            throw new IllegalArgumentException('error message')
         } should throwException(UnsupportedOperationException)
     }
 
     @Test
     void "should validate exception class and expected message pattern"() {
-        thrown.expectMessage("\nmismatches:\n" +
-                "\n" +
-                "expected exception message:    actual string: error message\n" +
-                "                            expected pattern: error \\d\n" +
-                "expected exception class:   actual: class java.lang.IllegalArgumentException <java.lang.Class>\n" +
-                "                          expected: class java.lang.UnsupportedOperationException <java.lang.Class>")
+        thrown.expectMessage('\nmismatches:\n' +
```

Comment:
What's with all the `"` to `'` changes?

Reply:
I need to use " inside and also want to switch to using `'` by default.